Compilation issue for match_cell_barcode

I get the following error when trying to compile `match_cell_barcode`.

```sh
$ g++ -std=c++11 -lz -O2 -o match_cell_barcode ssw/ssw_cpp.cpp ssw/ssw.c match_cell_barcode.cpp kseq.h edit_dist.cpp
```
<details>
<summary>Error</summary>

```sh
edit_dist.cpp: In function ‘unsigned int edit_distance_bpv(T&, const int64_t*, const size_t&, const unsigned int&, const unsigned int&)’:
edit_dist.cpp:50:5: error: ‘uint64_t’ was not declared in this scope
   50 |     uint64_t top = (1LL << (tlen - 1));
      |     ^~~~~~~~
edit_dist.cpp:2:1: note: ‘uint64_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
    1 | #include "edit_dist.h"
  +++ |+#include <cstdint>
    2 |
edit_dist.cpp:51:13: error: expected ‘;’ before ‘lmb’
   51 |     uint64_t lmb = (1LL << 63);
      |             ^~~~
      |             ;
edit_dist.cpp:62:21: error: expected ‘;’ before ‘X’
   62 |             uint64_t X = PM[r];
      |                     ^~
      |                     ;
edit_dist.cpp:63:38: error: ‘lmb’ was not declared in this scope
   63 |             if(r > 0 && (HN[r - 1] & lmb)) X |= 1LL;
      |                                      ^~~
edit_dist.cpp:63:44: error: ‘X’ was not declared in this scope                                                                                        [94/1081]
   63 |             if(r > 0 && (HN[r - 1] & lmb)) X |= 1LL;
      |                                            ^
edit_dist.cpp:64:24: error: ‘X’ was not declared in this scope
   64 |             D0[r] = (((X & VP[r]) + VP[r]) ^ VP[r]) | X | VN[r];
      |                        ^
edit_dist.cpp:68:38: error: ‘lmb’ was not declared in this scope
   68 |             if(r == 0 || HP[r - 1] & lmb) X |= 1LL;
      |                                      ^~~
edit_dist.cpp:70:38: error: ‘lmb’ was not declared in this scope
   70 |             if(r > 0 && (HN[r - 1] & lmb)) VP[r] |= 1LL;
      |                                      ^~~
edit_dist.cpp:73:23: error: ‘top’ was not declared in this scope
   73 |         if(HP[tmax] & top) ++D;
      |                       ^~~
edit_dist.cpp: At global scope:
edit_dist.cpp:82:5: error: ‘uint64_t’ does not name a type
   82 |     uint64_t arr_[N];
      |     ^~~~~~~~
edit_dist.cpp:82:5: note: ‘uint64_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
edit_dist.cpp:83:5: error: ‘uint64_t’ does not name a type
   83 |     uint64_t & operator[](size_t const &i) {
      |     ^~~~~~~~
edit_dist.cpp:83:5: note: ‘uint64_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
edit_dist.cpp: In instantiation of ‘unsigned int edit_distance_map_(const int64_t*, size_t, const int64_t*, size_t) [with long unsigned int N = 1; int64_t = lo
ng int; size_t = long unsigned int]’:
edit_dist.cpp:113:48:   required from here
edit_dist.cpp:95:59: error: no match for ‘operator[]’ (operand types are ‘std::map<long int, varr<1>, std::less<long int>, std::allocator<std::pair<const long
int, varr<1> > > >::mapped_type’ {aka ‘varr<1>’} and ‘size_t’ {aka ‘long unsigned int’})
   95 |         for(size_t j = 0; j < 64; ++j) cmap[a[i * 64 + j]][i] |= (1LL << j);
      |                                        ~~~~~~~~~~~~~~~~~~~^
edit_dist.cpp:97:60: error: no match for ‘operator[]’ (operand types are ‘std::map<long int, varr<1>, std::less<long int>, std::allocator<std::pair<const long
int, varr<1> > > >::mapped_type’ {aka ‘varr<1>’} and ‘unsigned int’)
   97 |     for(size_t i = 0; i < tlen; ++i) cmap[a[tmax * 64 + i]][tmax] |= (1LL << i);
      |                                      ~~~~~~~~~~~~~~~~~~~~~~^
edit_dist.cpp: In instantiation of ‘unsigned int edit_distance_map_(const int64_t*, size_t, const int64_t*, size_t) [with long unsigned int N = 2; int64_t = lo
ng int; size_t = long unsigned int]’:
edit_dist.cpp:114:53:   required from here
edit_dist.cpp:95:59: error: no match for ‘operator[]’ (operand types are ‘std::map<long int, varr<2>, std::less<long int>, std::allocator<std::pair<const long
int, varr<2> > > >::mapped_type’ {aka ‘varr<2>’} and ‘size_t’ {aka ‘long unsigned int’})
   95 |         for(size_t j = 0; j < 64; ++j) cmap[a[i * 64 + j]][i] |= (1LL << j);
      |                                        ~~~~~~~~~~~~~~~~~~~^
edit_dist.cpp:97:60: error: no match for ‘operator[]’ (operand types are ‘std::map<long int, varr<2>, std::less<long int>, std::allocator<std::pair<const long
int, varr<2> > > >::mapped_type’ {aka ‘varr<2>’} and ‘unsigned int’)
   97 |     for(size_t i = 0; i < tlen; ++i) cmap[a[tmax * 64 + i]][tmax] |= (1LL << i);
      |                                      ~~~~~~~~~~~~~~~~~~~~~~^
edit_dist.cpp: In function ‘unsigned int scutil::edit_distance1(const int64_t*, unsigned int, const int64_t*, unsigned int)’:
edit_dist.cpp:115:1: warning: control reaches end of non-void function [-Wreturn-type]
  115 | }
      | ^
```

</details>

Here's some info about my environment, if it's relevant.

<details>
<summary>Environment information</summary>

```sh
$ g++ --version
g++ (GCC) 13.2.0
Copyright (C) 2023 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

$ ldd --version
ldd (GNU libc) 2.28
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
Written by Roland McGrath and Ulrich Drepper.

$ uname -a
Linux hostname 4.18.0-477.13.1.el8_8.x86_64 #1 SMP Tue May 30 22:15:39 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
```

</details>

--- a/src/edit_dist.h
+++ b/src/edit_dist.h
@@ -4,6 +4,7 @@
 #include <bitset>
 #include <algorithm>
 #include <stdlib.h>
+#include <cstdint>
 
 namespace scutil{
     int hamming_distance(const std::string &A, const std::string &B);

